File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL3010U00DsvUrineListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL3010U00DsvUrineListItemInfo
	{
		private String _pkcpl1000;
		private String _urineRyang;
		private String _height;
		private String _weight;
		private String _stabilityYn;
		private String _specimenSer;

		public String Pkcpl1000
		{
			get { return this._pkcpl1000; }
			set { this._pkcpl1000 = value; }
		}

		public String UrineRyang
		{
			get { return this._urineRyang; }
			set { this._urineRyang = value; }
		}

		public String Height
		{
			get { return this._height; }
			set { this._height = value; }
		}

		public String Weight
		{
			get { return this._weight; }
			set { this._weight = value; }
		}

		public String StabilityYn
		{
			get { return this._stabilityYn; }
			set { this._stabilityYn = value; }
		}

		public String SpecimenSer
		{
			get { return this._specimenSer; }
			set { this._specimenSer = value; }
		}

		public CPL3010U00DsvUrineListItemInfo() { }

		public CPL3010U00DsvUrineListItemInfo(String pkcpl1000, String urineRyang, String height, String weight, String stabilityYn, String specimenSer)
		{
			this._pkcpl1000 = pkcpl1000;
			this._urineRyang = urineRyang;
			this._height = height;
			this._weight = weight;
			this._stabilityYn = stabilityYn;
			this._specimenSer = specimenSer;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL3010U00DsvUrineListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL3010U00DsvUrineListItemInfo {
	private Double pkcpl1000;
    private String urineRyang;
    private String height;
    private String weight;
    private String stabilityYn;
    private String specimenSer;
	public CPL3010U00DsvUrineListItemInfo(Double pkcpl1000, String urineRyang,
			String height, String weight, String stabilityYn, String specimenSer) {
		super();
		this.pkcpl1000 = pkcpl1000;
		this.urineRyang = urineRyang;
		this.height = height;
		this.weight = weight;
		this.stabilityYn = stabilityYn;
		this.specimenSer = specimenSer;
	}
	public Double getPkcpl1000() {
		return pkcpl1000;
	}
	public void setPkcpl1000(Double pkcpl1000) {
		this.pkcpl1000 = pkcpl1000;
	}
	public String getUrineRyang() {
		return urineRyang;
	}
	public void setUrineRyang(String urineRyang) {
		this.urineRyang = urineRyang;
	}
	public String getHeight() {
		return height;
	}
	public void setHeight(String height) {
		this.height = height;
	}
	public String getWeight() {
		return weight;
	}
	public void setWeight(String weight) {
		this.weight = weight;
	}
	public String getStabilityYn() {
		return stabilityYn;
	}
	public void setStabilityYn(String stabilityYn) {
		this.stabilityYn = stabilityYn;
	}
	public String getSpecimenSer() {
		return specimenSer;
	}
	public void setSpecimenSer(String specimenSer) {
		this.specimenSer = specimenSer;
	}
}
